colour/colorimetry/illuminants.py

```diff
@@ -80,3 +86,42 @@ def D_illuminant_relative_spd(xy):
 
     return SpectralPowerDistribution('CIE Standard Illuminant D Series',
                                      distribution)
+
+
+def CIE_standard_illuminant_A(wl):
```

Comment:
Yes! I wasn't really happy with the name either. I was thinking about something like `CIE_standard_illuminant_A_function` otherwise as we have definitions like `mesopic_weighting_function` or `mesopic_luminous_efficiency_function`.

I have a few other naming convention points I'll be keen to discuss too, maybe on Slack or a dedicated issue.